docs/docfx.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://raw.githubusercontent.com/dotnet/docfx/main/schemas/docfx.schema.json",
+  "metadata": [
+    {
+      "src": [
+        {
+          "src": "../src",
+          "files": [
+            "**/*.csproj"
+          ]
+        }
+      ],
+      "dest": "api"
+    }
+  ],
+  "build": {
+    "content": [
+      {
+        "files": [
+          "**/*.{md,yml}"
+        ],
+        "exclude": [
+          "_site/**"
+        ]
+      }
+    ],
+    "resource": [
+      {
+        "files": [
+          "images/**"
+        ]
+      }
+    ],
+    "output": "_site",
+    "template": [
+      "default",
+      "modern"
+    ],
+    "globalMetadata": {
+      "_appName": "MCP .NET SDK",
+      "_appTitle": "MCP .NET SDK",
+      "_enableSearch": true,
+      "pdf": true,
+      "_appLogoPath": "images/logo.png",
```

Comment:
The default docfx logo is in svg, might make sense to use the same format here.

Reply:
I used this to create the main package logo: https://github.com/modelcontextprotocol/docs/blob/main/favicon.svg

Reply:
I can swap it out.